### PR TITLE
Skip tour

### LINKED ISSUE #727: Let the Tour have "Skip" and "Next" buttons
This is confusing

image

### PR DESCRIPTION
**Pull request by AmirSavand**
Resolves #727 

Skipping a tour will skip the current tour only not all the tours in every view.

image

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,3 +6,8 @@
 - Resolved #242: Add a filter to order tags based on how many times they have been used.
 - Resolved #787: Display updated by 'user' at Entry grid view.
 - Resolved #886: Display blog followers in the main view.
+
+2018.7.2
```
Comment (Alir3z4) [inline CHANGELOG.md]: The new release should be upper.
The olders go below.

File: src/app/header/header_view.html [hunk 2]
```diff
@@ -106,11 +106,14 @@
 <!-- Popover -->
 <div class="popover tour animated">
   <div class="arrow"></div>
-  <button class="btn btn-success" ng-click="tour.nextStep()">
-    <i class="fa fa-check"></i>
-  </button>
   <div class="popover-title"></div>
   <div class="popover-content"></div>
+  <div class="popover-footer">
+    <button class="btn btn-default" ng-click="tour.skip()">Skip</button>
```
Comment (Alir3z4) [inline src/app/header/header_view.html]: Needs translation.

File: src/app/header/header_view.html [hunk 3]
```diff
@@ -106,11 +106,14 @@
...
+    <button class="btn btn-success" ng-click="tour.nextStep()">
+      <i class="fa fa-check"></i> Next
```
Comment (Alir3z4) [inline src/app/header/header_view.html]: Also need translation

**Comment by Alir3z4** [comment]
Great work here.
This is much better now.

I requested some changes.
Please address them and we're good to go

File: CHANGELOG.md (hunk 1 above)
Comment (AmirSavand) [reply CHANGELOG.md]: oh yea

**Comment by AmirSavand** [comment]
Yes.

**Pull request merged**